diff --git a/node_modules/@vueuse/motion/dist/nuxt.mjs b/node_modules/@vueuse/motion/dist/nuxt.mjs
index c885c58..7f7c5b7 100644
--- a/node_modules/@vueuse/motion/dist/nuxt.mjs
+++ b/node_modules/@vueuse/motion/dist/nuxt.mjs
@@ -1,4 +1,4 @@
-import { defineNuxtModule, createResolver, addPlugin, addAutoImportDir, resolveModule } from '@nuxt/kit';
+import { defineNuxtModule, createResolver, addPlugin, addImportsDir, resolveModule } from '@nuxt/kit';
 
 const DEFAULTS = {};
 const CONFIG_KEY = "motion";
@@ -11,7 +11,7 @@ const module = defineNuxtModule({
   setup(options, nuxt) {
     const { resolve } = createResolver(import.meta.url);
     const resolveRuntimeModule = (path) => resolveModule(path, { paths: resolve("./runtime") });
-    nuxt.options.publicRuntimeConfig.motion = options;
+    nuxt.options.runtimeConfig.public.motion = options;
     addPlugin({
       src: resolveRuntimeModule("./templates/motion.js")
     });
@@ -26,7 +26,7 @@ const module = defineNuxtModule({
       nuxt.options.alias = {};
     if (!nuxt.options.alias.tslib)
       nuxt.options.alias.tslib = "tslib/tslib.es6.js";
-    addAutoImportDir(resolve("./composables/"));
+    addImportsDir(resolve("./composables/"));
   }
 });
 